### PR TITLE
Add ember-source 5 support

### PR DESCRIPTION
**Pull request by SergeAstapov**
credit for majority of work goes to @hexadecy

### CODE REVIEW

**Comment by SergeAstapov** [comment]
@elwayman02 could you release this as a minor 7.2.0?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -54,8 +54,8 @@ jobs:
         try-scenario:
           - ember-lts-3.28
           - ember-lts-4.4
+          - ember-lts-4.8
           - ember-release
-          - ember-classic
           - embroider-safe
```
Comment (elwayman02) [inline .github/workflows/ci.yml:59]: Now that we have embroider-optimized passing, can you move it out of the experimental group and into the required list with embroider-safe here?

Comment (SergeAstapov) [reply .github/workflows/ci.yml:59]: good point! Done!

**Review by elwayman02 — APPROVED** [review]

**Comment by elwayman02** [comment]
Thanks @hexadecy @SergeAstapov for getting this done! I'll get the release out tomorrow when I'm back at a terminal.

**Comment by elwayman02** [comment]
@SergeAstapov 7.2 is released!

**Pull request merged**